source/PluginDev/Assets/GooglePlayGames/Editor/GPGSStrings.cs

```diff
@@ -95,6 +95,10 @@ public class AndroidSetup
             public const string Blurb = "To configure Google Play Games in this project,\n" +
                                         "please enter the information below and click on the Setup button.";
 
+            public const string ClientIdBlurb = "Enter your oauth2 client ID below. To obtain this\n" +
```

Comment:
They are different things, 70-72 is mentioning iOS, 98-100 are Android.  I'm more than happy to change to something else, but I think they may need slightly different descriptions for what they're logically referring to.